src/fetch/period_slippage.py

```diff
@@ -5,16 +5,43 @@
 from src.dune_analytics import DuneAnalytics, QueryParameter
 from src.file_io import File
 from src.models import Network
+from src.token_list import HOSTED_ALLOWED_BUFFER_TRADING_TOKEN_LIST_URL, get_trusted_tokens_from_url
+
+
+def generate_sql_query_for_allowed_token_list(token_list) -> str:
+    query = "allow_listed_tokens as ( Select * from (VALUES"
+    for address in token_list:
+        query = "".join([query, f"('\\{address[1:]}' :: bytea),"])
+    query = query[:-1]
```

Comment:
Why this slice?

Reply:
We need to take out the last comma

Reply:
To eliminate the need to remove the last comma (and iterate over the list). consider the following:

```py
values = ",".join(f"('\\{address[1:]}' :: bytea)," for address in token_list)
query = f"allow_listed_tokens as (select * from (VALUES {values}) AS t (token)),"
```